framework/src/main/java/org/checkerframework/framework/type/AnnotatedTypeFactory.java

```diff
@@ -1204,6 +1170,7 @@ public AnnotatedTypeMirror getTypeOfExtendsImplements(Tree clause) {
     // They only include qualifiers explicitly inserted by the user.
     // **********************************************************************
 
+    StubTypes stubTypes = new StubTypes(this);
```

Comment:
It's a bit hard to see in the diff... Is this a field? Why does it have default visibility? Can it be final? Why is it down here and not with other fields at the beginning?